src/main/java/Ren/task/Event.java

```diff
@@ -0,0 +1,61 @@
+package ren.task;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+/**
+ * Represents an event
+ */
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class Event extends Task {
+
+    private final String START_TIME;
+    private final String END_TIME;
+
+    public String getDescription() {
+        return this.description;
+    }
+
+    public boolean getIsDone() {
+        return this.isDone;
+    }
+
+    public TaskType getTaskType() {
+        return this.taskType;
+    }
+
+    public String getSTART_TIME() {
+        return this.START_TIME;
+    }
+
+    public String getEND_TIME() {
+        return this.END_TIME;
+    }
```

Comment:
Could the tell-don't-ask OOP principle be employed here instead of getter methods ?